packages/aila/src/features/generation/AilaGeneration.ts

```diff
@@ -139,16 +139,20 @@ export class AilaGeneration {
   }
 
   private async calculateTokenUsage(): Promise<void> {
```

Comment:
If we remove the `async` declaration here then I imagine we don't need to wrap in a promise?

Reply:
I didn't want to change the behaviour itself. I think because we originally had another method for calculating tokens and it might be that we need to keep this async

Reply:
Got it. Looking at the interface I didn't see any other implementations that would be async, but I didn't think of that